packages/cli/templates/platform-app-tailwind/src/App.tsx

```diff
@@ -0,0 +1,120 @@
+import React, {useEffect} from 'react';
+import frontifyLogo from './assets/frontify.svg'
+import {
+    Button,
+    ButtonEmphasis,
+    Card,
+    LoadingCircle,
+    LoadingCircleSize,
+    LoadingCircleStyle,
+    Stack,
+    Text
+} from "@frontify/fondue";
+import {usePlatformAppBridge} from "./usePlatformAppBridge";
+import {PlatformAppContext} from "@frontify/app-bridge";
+
+
+type AssetResource = {
+    type: string;
+    id: string;
+    title?: string;
+    previewUrl?: string;
+    downloadUrl?: string;
+    filename?: string | null;
+    sourceUrl?: string;
+    html?: string;
+}
+
+const App = () => {
+    const appBridge = usePlatformAppBridge();
+
+    const [imageSrc, setImageSrc] = React.useState<string | undefined>(undefined);
+    const [assetRessources, setAssetRessources] = React.useState<AssetResource>();
+    const [context, setContext] = React.useState<PlatformAppContext>();
+    const [reUploadCounter, setReUploadCounter] = React.useState<number>(0);
+    const [uploading, setUploading] = React.useState<boolean>(false);
+
+    useEffect(() => {
+        (async () => {
+            if (!appBridge) return;
+
+            const currentContext = appBridge.context().get();
+            setContext(currentContext);
+
+            if (currentContext?.type === "ASSET_ACTION") {
+
+                const assetRessources = await appBridge.api({
+                    name: 'getAssetResourceInfo',
```

Comment:
```suggestion
                    name: 'getAssetResourceInformation',
```
Please update the app-bridge to not have abbreviation